SpatialGDK/Documentation/content/non-unreal-server-worker-types.md

```diff
@@ -39,20 +39,22 @@ There is a basic example in the _Examples_ section below. For more examples of h
 
 >**Note:** Your external SpatialOS components must have an ID between 1000 and 2000 to be registered by the pipeline.
 
-You set up your game to receive component updates through implementing callbacks for specified SpatialOS component IDs and operation types. The GDK detects your callbacks when it’s initialized and uses them for all received data relating to components with IDs in the range 1000 to 2000.
-To create callbacks, you need to create a class derived from the GDK `UOpCallbackTemplate` base class and implement `GetComponentId` and one or more of the callback methods. These callback methods are parameterized by the following network operation types:
+You set up your game to receive network operations using the `USpatialDispatcher::AddOpCallback` functions. These overloaded functions are parameterized with a `Worker_ComponentId` and a callback function reference that takes one of the following network operation types as an argument:
 
-* `virtual void OnAddComponent(const Worker_AddComponentOp& Op) {}`
-* `virtual void OnRemoveComponent(const Worker_RemoveComponentOp& Op) {}`
-* `virtual void OnComponentUpdate(const Worker_ComponentUpdateOp& Op) {}`
-* `virtual void OnAuthorityChange(const Worker_AuthorityChangeOp& Op) {}`
-* `virtual void OnCommandRequest(const Worker_CommandRequestOp& Op) {}`
-* `virtual void OnCommandResponse(const Worker_CommandResponseOp& Op) {}`
+* `Worker_AddComponentOp`
+* `Worker_RemoveComponentOp`
+* `Worker_AuthorityChangeOp`
+* `Worker_ComponentUpdateOp`
+* `Worker_CommandRequestOp`
+* `Worker_CommandResponseOp`
 
+`Worker_ComponentId` and each network operation type are defined in the [Worker SDK in C’s API](https://docs.improbable.io/reference/latest/capi/api-reference).
 
-The `UOpCallbackTemplate` base class contains protected references to the `UWorld` and GDK `StaticComponentView` enabling you to write more meaningful callbacks.
+You must register the callbacks inside your game instance's `::Init()` function to ensure your callbacks will be triggered for initial network operations received by the SpatialOS worker connection.
```

Comment:
nit: This reads like you *must* do it for all callbacks, when really it's only if you might receive ops on initial connection.